tests/testhelpers/generators.go

```diff
@@ -17,13 +17,15 @@ func NewRand() *rand.Rand {
 // Test Helpers that are useful for Generating random Scoot Api Structs
 // To help with testing the Scoot API and Scheduler
 
-// Generates a scoot.JobDefinition
-func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
-
+// Generates a scoot.JobDefinition with numTasks tasks (or random if numTasks == -1)
```

Comment:
Less than 1B? Actually it would be good to know how much is too much on a macbook.